atlp-rwanda/rca-barefoot-team4-bn
# #184638221 integrate Github actions with readme badge

What does this PR do?
=>This PR has an updated readme file that contains a badge

Description of Task to be completed
=> I have added markdown code for the build action in the readme file check if the badge appears successfully. Once the readme badge is successful I can add more GitHub actions depending on the project actions.

How should this be manually tested?
=> This can be tested on the GitHub repo by checking if a badge appear on top of the displayed texts from readme file

Any background context you want to provide?
=> No background context

What are the relevant pivot tracker story?
=> This is required to automatically run tests and other preliminary build steps required to ship the next iteration of the product

Screenshots (if appropriate) no screenshots
